lib/tidalcycles.js

```diff
@@ -49,12 +50,25 @@ export default {
         description: `Console prompt. Look at the docs for available placeholders`,
         order: 50
       },
+
       'showErrorNotifications': {
         type: 'boolean',
         default: true,
         description: 'Show atom notifications on error.',
         order: 55
       },
+      'oscEvalPort': {
+        type: 'integer',
+        default: 3333,
+        description: `OSC port for receiving eval messages.`,
+        order: 56
```

Comment:
We can set order index 80 and 81 to oscEvalPort and oscEvalAddress, to better distribute the configuration properties

Reply:
I changed the order number to 80, 81 and 82 and add the property "oscEvalIp".